src/styles/_variables.scss

```diff
@@ -38,4 +40,7 @@ $secondary-hover: #682C73;
 $warning-hover: #D98E25;
 $alert-hover: #D93B30;
 
-$third: rgb(130,153,156);
\ No newline at end of file
+$third: rgb(130,153,156);
+
+// fonts
+$heading-font: 'Noto Sans JP', sans-serif;
```

Comment:
if you're positive this font is only being used for the heading, great

if it is going to be used for all of Treobytes application, a more suitable name is just font-family

Reply:
font-family can be used. the wireframe uses 'Franklin Goth Book', 'Franklin Gothic Demi Cond', and 'Franklin Gothic Heavy'. CSS seems to have 'Franklin Gothic', but I could not get the Demi version. Google Fonts has a similar font called 'Libre Franklin'. I decided to use 'Libre Franklin'.